README.md

```diff
@@ -1,2 +1,17 @@
 # sourcerer-app
 Sourcerer app makes engineering profiles from their git repositories.
+
+Install/uninstall
+=================
+
+Run the command to install `sourcerer` tool on your machine:
+
+```
+curl -s https://sourcerer.io/install | sudo bash
+```
+
+Run the command to remove it from your machine:
+
+```
+curl -s https://sourcerer.io/uninstall | sudo bash
```

Comment:
I've got used to think (in OS X world) uninstallation process is that simple as removing the app itself (which is a folder technically). In Linux world people do apt-get remove. It seems neither of these options works in this case. So what name and location for this script could be?